SpatialGDKEditorToolbar/Source/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -138,16 +138,58 @@ void WriteSchemaRPCField(TSharedPtr<FCodeWriter> Writer, const TSharedPtr<FUnrea
 	);
 }
 
+// core_types.schema should only be included if any components in the file have
+// 1. An UnrealObjectRef
+// 2. A list of UnrealObjectRefs
+// 3. A RPC
```

Comment:
Nit: `An RPC`

Reply:
I'm pretty sure this doesn't follow English rules